webplatform/config/initializers/gibbon.rb

```diff
@@ -0,0 +1,3 @@
+Gibbon::Request.api_key = "f677a695c9cf2688302922eb90c0eb2c-us11"
```

Comment:
it would be good to use `dotenv` gem here so we're not putting our api keys and credentials directly in the code. 
To do that add this gem to `Gemfile` and run `bundle`. Please create 2 files - .env.template and .env (in the root dir so in the same place as Gemfile for example). Let's then add .env to .gitignore. Now in .env.template let's put only e.g `GIBBON_KEY=` and in your .env put `GIBBON_KEY=f677a695c9cf2688302922eb90c0eb2c-us11` and now in webplatform/config/initializers/gibbon.rb instead of explicitly putting a key, you can write only `Gibbon::Request.api_key = ENV["GIBBON_KEY"]`. Obviously we'll need to put real key in ticket in asana :)